tests/tests/test_configuration.py

```diff
@@ -164,6 +164,9 @@ def was_update_forced(mender_device):
     elif "Cannot check update or update inventory while in update-fetch state" in out:
         # race condition - check-update came while we were already updating
         return False
+    elif "/usr/share/mender/modules/v3/mender-configure" in out:
+        # deployment was processed too quickly
```

Comment:
could you explain that to me? I mean: how can this happen?

Reply:
The device is slow, starts when the deployment is already pending; therefore, it picks up the deployment in a natural way without the push from mender-connect. It will output "mender-configure" in the logs because it processes the configuration deployment, and we return "False" because there was no push. Before this fix, it raised an Exception and failed without retrying.